### ISSUE TITLE
InlineNotification is not read out by screen reader in FireFox

### ISSUE THREAD

**Issue by amuqeetkhan** [carbon-design-system/carbon#12345]
- The InlineNotification content sometimes cannot be read out by screen reader on Firefox
- It works fine with chrome

**Comment by tay1orjones** [comment]
@amuqeetkhan Next time please use the [accessibility issue template](https://github.com/carbon-design-system/carbon/issues/new?assignees=&labels=type%3A+a11y+%E2%99%BF&template=ACCESSIBILITY_ISSUE.yaml&title=%5Ba11y%5D%3A+) we have available to provide the full detail we need to investigate and fix the issue.

Most importantly, could you provide the following additional info:
- What package are you using?
- What version of the package are you using?
- What screenreader were you using?
- Is this reproducible on the storybook? If not, please provide a [codesandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/v11.15.0/examples/codesandbox) reproduction link

**Comment by sstrubberg** [comment]
+1 to @tw15egan's suggestion. Let us know and we'll be happy to re-open the issue.
